rust/core-lib/src/tabs.rs

```diff
@@ -986,7 +986,7 @@ impl<'de> Deserialize<'de> for ViewId {
         D: Deserializer<'de>,
     {
         let s = String::deserialize(deserializer)?;
-        let ord = s.trim_left_matches("view-id-");
```

Comment:
Was this causing a CI failure? it feels like maybe it should be a separate patch?

Reply:
Yeah, this is causing the nightly build to fail since `-D warnings` is set. Coverage only runs on nightly, so I fixed it to at least attempt the coverage run.